docs/monitoring/autodiscovery.md

```diff
@@ -90,6 +90,30 @@ In container environments you add the `MONITORING_TOKEN` environment variable, o
 
 In all environments this makes it possible to use different monitoring tokens, and thus different Sematext Apps for the same service type running on the same machine or in the same Kubernetes/Swarm cluster.
 
+### Switching from manual to automatic monitoring
+
+The switch from manual to automatic monitoring is done by first removing manually set up agent and then enabling automatic monitoring in the user interface.
+
+In bare-metal/virtual machine setups there are two types of agents - in-process (with javaagent) and standalone.
+
+If you are using in-process variant, do the following:
+- remove `-javaagent` definition from the startup script of your service and restart the service
+- from the directory `/opt/spm/spm-monitor/conf`, remove the config file whose name contains an App token used by this specific service
+
+If you are using standalone variant:
+- from the directory `/opt/spm/spm-monitor/conf`, remove the config file whose name contains an App token used by this specific service
+- restart the agent with `sudo service sematext-agent restart
+
+In container environments it is much simpler - just remove environment variable named MONITORING_TOKEN from your containers:
```

Comment:
In container environments it is much simpler - just remove environment variable named MONITORING_TOKEN from your containers

->

In container environments it's much simpler - remove the environment variable named `MONITORING_TOKEN` from your containers